catalogs/iceberg-hadoop/README.md

```diff
@@ -75,5 +75,5 @@ Example output:
 ## Step 4: Cleanup.
 
 ```bash
-docker-compose down
+docker compose down --volumes --rmi local
```

Comment:
`docker-compose`->`docker compose` is changed intentionally to be consistent with other places including `docker compose up` earlier in the same recipe